sdk/storage/storage-blob/src/AppendBlobClient.ts

```diff
@@ -312,19 +324,21 @@ export class AppendBlobClient extends BlobClient {
             extractedCreds.accountName!,
             extractedCreds.accountKey
           );
-          urlOrConnectionString = extractedCreds.url + "/" + containerName + "/" + blobName;
+          url = appendToURLPath(
+            appendToURLPath(extractedCreds.url, encodeURIComponent(containerName)),
```

Comment:
Minor: Would this better be endpoint? ([https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string))

Reply:
Yeah, makes sense.
However, there are many instances of it. And this won't be a breaking change since it is internal to the SDK. Will do it post preview 4.

Reply:
#5460